core/templates/domain/exploration/exploration-id-validation.service.ts

```diff
@@ -16,19 +16,31 @@
  * @fileoverview Service to validate exploration id
  */
 
-require('domain/summary/exploration-summary-backend-api.service.ts');
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
```

Comment:
The import order we follow is this:

1. Angular imports (imports from @angular/*)
2. Third-party imports (imports from rxjs, ng bootstrap, etc)
3. Application imports. (imports from our application)

Within these sub-categories: sort lexicographically based on module (i.e. where it is imported from, not what is being imported).
For example:
#### Right way (because 'B' < 'D')
```
import { C } from 'B';
import { A } from 'D'; 
```

#### Wrong way
```
import { A } from 'D';
import { C } from 'B'; 
```
```suggestion
import { Injectable } from '@angular/core';
import { downgradeInjectable } from '@angular/upgrade/static';
```

Reply:
done!